### PR TITLE
New bbl subjects

### PR DESCRIPTION
**Pull request by Fabinout**

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -14768,8 +14796,7 @@ var data = {
       "name": "Jean Bisutti",
       "bio": "Développeur Java",
       "picture": "https://pbs.twimg.com/profile_images/606213307507351552/IJcDzWZo.jpg",
-      "websites": [
```
Comment (dadoonet) [inline baggers.js]: This is not needed. Could you revert that change?

**Review by dadoonet — CHANGES_REQUESTED** [review]
Left a minor comment. Could you update?

File: baggers.js (hunk 1 above)
Comment (Fabinout) [reply baggers.js]: right away !

**Comment by Fabinout** [comment]
Fixed

**Review by dadoonet — APPROVED** [review]

**Comment by caedes** [comment]
Une virgule a été oublié au moment du dév et à la review.

D'où le bug suivant :
bug

File: baggers.js [hunk 2]
```diff
@@ -12651,14 +12651,42 @@ var data = {
...
+            "Code Review",
+            "Agile",
+            "XP",
+            "Software Craftsmanship",
+            "team"
+          ]
+        },
+        {
+          "title": "Introduction à git",
+          "abstract": "Atelier d'apprentissage de git. On commence par une brève introduction aux principes, puis on s'essaie à un atelier pratique pour apprendre les commandes basiques."
```
Comment (dadoonet) [inline baggers.js:12667]: Argh! Comma missing indeed!

Comment (dadoonet) [reply baggers.js:12667]: Exact @caedes. Thanks!

**Comment by dadoonet** [comment]
Fixed. Thanks @caedes for reporting.

**Comment by caedes** [comment]
Cool! Thx

~What about if I provide a better system to handle all this data? I thought about combining multi JS files, one per speaker, into `baggers.js` build with [Parcel](https://parceljs.org/). Nearly have the PR.~

Forget about it, just see #160

**Pull request merged**